packages/components/src/dropdown/stories/index.tsx

```diff
@@ -21,17 +26,24 @@ export default {
 		renderContent: { control: { type: null } },
 		renderToggle: { control: { type: null } },
 	},
+	parameters: {
+		controls: {
+			expanded: true,
+		},
+		docs: { source: { state: 'open' } },
```

Comment:
```suggestion
		controls: {
			expanded: true,
		},
```

I guess we should keep the expanded controls, but remove the open docs since the code snippets aren't very helpful in this case?

Reply:
Good idea. Committed in https://github.com/WordPress/gutenberg/pull/45787/commits/775f10b1de7fb5faf0c3ab3cf9588496c672f078